hardware_interface/source/hyperion.c

```diff
@@ -70,64 +72,56 @@ void hyperion_config_1_value(config_1_panel_t panel, config_1_channel_type_t cha
     /* Channel 1 to 3 handle temp sensor */
     case CONFIG_1_CHANNEL_TEMP_1: {
         adc_init(slave_addr, ADC_CHANNEL_0);
-        adc_get_raw(slave_addr, &data, &ch); // invalid value
-        adc_get_raw(slave_addr, &data, &ch); // correct value
+        adc_get_raw(slave_addr, &data, &ch);
```

Comment:
Did something change about how the adc works?

Reply:
It never needed to be checked twice afaik